index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Pat's Salmon Cookies</title>
+    <nav></nav>
```

Comment:
If this nav tag is empty, remove it. Generally, nav tags don't go in `<head>`, but in `<body>` so that their contents can actually be seen.